### PR TITLE
Fix bug with datepicker in Safari

### PR DESCRIPTION
**Pull request by cincospenguinos**
[Taman shud](https://www.pivotaltracker.com/story/show/175603008)

## PR upkeep checklist

- [x] Change log entry
- [x] Label(s)
- [x] Assignee(s)
- [x] Deployment URL: https://mavenlink.github.io/design-system/webinf-hey-everybody-get-on-down-with-the-dateness-175603008/
- [x] (When ready for review) Reviewer(s)
- [x] Green Bigmaven CI check
- [ ] DevQA on Bigmaven staging (e.g. does the work need to be imported in the internal design system?)

### CODE REVIEW

File: src/components/custom-field-input-date/custom-field-input-date.jsx [hunk 1]
```diff
@@ -37,7 +37,14 @@ function presentDateString(dateString) {
   }
 
   const timezone = new Date().toLocaleTimeString('en-us', { timeZoneName: 'short' }).split(' ')[2];
-  const date = new Date(`${dateString}  00:00:00 ${timezone}`);
+  let date = new Date(`${dateString}  00:00:00 ${timezone}`);
+
+  if (date.toString() === 'Invalid Date') {
+    // Okay, we're in Safari and we need to find a way to parse the date here
+    const split = dateString.split('-');
+    date = new Date(Number(split[0]), Number(split[1]) - 1, Number(split[2]));
```
Comment (bennapp) [inline src/components/custom-field-input-date/custom-field-input-date.jsx:45]: Just curious, why `Number(split[1]) - 1` here?

**Review by bennapp — APPROVED** [review]
LGTM

File: src/components/custom-field-input-date/custom-field-input-date.jsx (hunk 1 above)
Comment (juanca) [reply src/components/custom-field-input-date/custom-field-input-date.jsx:45]: Date takes many arguments:

1. year
2. month (0-based indicies)
3. date

...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
+  - Patch: Fix issue with custom field input date in Safari
```
Comment (juanca) [inline CHANGELOG.md:10]:
```suggestion
  - Patch: Fix issue with custom field input date in Safari, Firefox
```

**Pull request merged**